runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationSections.module.scss

```diff
@@ -0,0 +1,160 @@
+@import "#src-landing/scss/main.scss";
+
+$ARROW_ICON_WIDTH: 300px;
+
+.root {
+  background-color: $palette-accent-primary;
+  position: relative;
+}
+
+.arrowIconWrapper {
+  width: $ARROW_ICON_WIDTH;
+  position: absolute;
+}
+
+.arrowIcon {
+  width: 100%;
+  height: 100%;
+  filter: brightness(0) invert(1);
+}
+
+.centeredWrapper {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-top: $padding-10;
```

Comment:
you can replace it with single line
`padding: $padding-10 $padding-3;`

1st argument = bot & top, 2nd = right & left

Reply:
done